uint32 upTime
bool frontEStop
bool backEStop
bool wirelessEStop
bool bumpSensor
bool pause
bool hardStop
bool safetyProcessorFailure
bool brainstemFailure
bool brainTimeoutFailure
bool rightMotorFailure
bool leftMotorFailure
bool suspendState